src/olympia/amo/tests/__init__.py

```diff
@@ -1075,3 +1077,13 @@ def prefix_indexes(config):
 
     settings.CACHE_PREFIX = 'amo:{0}:'.format(prefix)
     settings.KEY_PREFIX = settings.CACHE_PREFIX
+
+
+def reverse_ns(viewname, api_version=None, args=None, kwargs=None, **extra):
```

Comment:
This needs some documentation. Especially mention that this generates a fake request on your behalf to resolve the URL.

Reply:
https://github.com/mozilla/addons-server/pull/8557/commits/faf93f2f080e6d5432010a938a85fdcf8d9acb25